SuperBuild/External_SimpleITK.cmake

```diff
@@ -177,7 +177,7 @@ ExternalProject_Execute(${proj} \"install\" \"${PYTHON_EXECUTABLE}\" setup.py in
       -DSimpleITK_PYTHON_THREADS:BOOL=ON
       -DSimpleITK_INSTALL_ARCHIVE_DIR:PATH=${Slicer_INSTALL_LIB_DIR}
       -DSimpleITK_INSTALL_LIBRARY_DIR:PATH=${Slicer_INSTALL_LIB_DIR}
-      -DSimpleITK_INT64_PIXELIDS:BOOL=OFF
+      -DSimpleITK_INT64_PIXELIDS:BOOL=ON
```

Comment:
Is there any commit or issue that could be referenced in the commit message to justify toggling this option ?

Reply:
Sure I can add reference to https://github.com/SimpleITK/SimpleITK/commit/2a9b7cdc2fa87b75ccbc9b6eeda8c1ff1b08e77f